src/components/RestaurantsList/RestaurantsList.scss

```diff
@@ -0,0 +1,5 @@
+.restaurant-list {
```

Comment:
restaurants-list
and also, what will happened when overflow? please set it to scroll